src/game/input/mod.rs

```diff
@@ -4,7 +4,7 @@ use crate::game::input::resource::BackspaceTimer;
 use crate::game::input::systems::*;
 
 mod systems;
-mod components;
+pub mod components;
```

Comment:
Undo?

Reply:
I think because you're using InputText component in word_match module you need it public